python/ml4ir/applications/ranking/model/ranking_model.py

```diff
@@ -21,6 +21,7 @@
 
 class RankingConstants:
     NEW_RANK = "new_rank"
+    KFOLD_METRICS = {'train_old_MRR', 'train_new_MRR', 'val_old_MRR', 'val_new_MRR', 'test_old_MRR', 'test_new_MRR'}
```

Comment:
@mohazahran As we discussed, I'm not sure why you are hardcoding this. You should create an argument and make these default at the least. How do you plan to make this work for classification?

Reply:
This might be a PITA comment: Can we consistently use `"` for strings?